data/events/scripts/creature.lua

```diff
@@ -7,6 +7,20 @@ function Creature:onAreaCombat(tile, isAggressive)
 end
 
 function Creature:onTargetCombat(target)
+	-- No Vocation Combat Config --
+	disableNoVocationCombat = true
```

Comment:
I don't think that's valid place to put this global variable in. Try moving to to config.lua and rename it to noVocationPvp = false, by default it should be false cause it was like that since ages.

Reply:
I like it within the script. It would just clutter config.lua, I don't think it is desireable that every possible customization is added to config.lua.